rails_console_commands.gemspec

```diff
@@ -23,5 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rubocop', '~> 0.49'
 
-  spec.add_dependency 'railties', '>= 5'
+  spec.add_dependency 'railties', '>= 7.2'
```

Comment:
Need to figure out a better way to do this so we can point to it, `7.2.0.alpha` is < `7.2`.

Reply:
Reverting for now so I can point to this to run tests locally.

Reply:
`7-2-stable` is pretty stable now, I think we can lock this in.

Reply:
https://github.com/TandaHQ/rails_console_commands/pull/2/commits/5feb5e456de85b92cb161d240341da51051deee7

Reply:
We can merge this if we want but need to be careful not to delete the branch cause payaus points to this. Might be safer to just leave as is though.